test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -237,6 +240,7 @@ class test_ucp_rma : public test_ucp_memheap {
                            rkey, param);
     }
 
+protected:
```

Comment:
Minor: It think it would be better to keep all the protected members in one place.